### PR TITLE
Removing the build directory from the repository

### PR DESCRIPTION
**Pull request by AndrewRayCode**

### CODE REVIEW

**Comment by ro31337** [comment]
Hey Andy, thanks for PR. But you know what, `build` directory is important. If you look at the "bin" entry https://github.com/ro31337/yaask/blob/master/package.json#L7, it says where to find the binary. In other words, something executable.

We can't point to `src/app.js`, because I used newer ES6 syntax which can be run on Node 6+ or 7+ something (guaranteed on [8.9.4, see .node-version](https://github.com/ro31337/yaask/blob/master/.node-version)). However, I wanted yaask to be compatible with older Node versions (see requirements in README and babelrc here https://github.com/ro31337/yaask/blob/master/.babelrc#L10). Newer syntax (ES6) won't work on older Node, that's why I used babel.

In other words, when you do `npm install yaask -g`, you will run yaask from build directory, and it can run on Node 4, 5, 6, 7, 8, etc., instead of just 8. It's bit late already, let me know if you have questions! Thanks for PR, I'll appreciate contribution and will take a look to other PRs tomorrow or a day after tomorrow.

**Comment by AndrewRayCode** [comment]
It should be fine to remove the build directory, because it still exists on disk, and that's what's published, even if it's not in git. npm publish pushes local specified files, not what's in git.

This _may_ require https://github.com/ro31337/yaask/pull/1/files with the files array, but I'm not sure.

I've made a few npm packages that work this way, where the files are built and then published, but never committed to source, and you simply build the files in a preublish step: https://github.com/AndrewRayCode/easing-utils/blob/master/package.json#L9

actually, I may be missing prepublish in this PR

**Comment by ro31337** [comment]
Nice, merging!
